test/Microsoft.DotNet.Docker.Tests/Dockerfile.linux.testapp.selfcontained

```diff
@@ -1,4 +1,5 @@
-FROM {base_image}
+ARG base_image
+FROM $base_image
```

Comment:
Clarification - `$base_image` vs. `${base_image}` what is the purpose of the curly brackets? Is there a risk by not including them. For example, I would prefer to have quotes around a variable that contains a path `"$programfiles%"`.

Reply:
$base_image is they way you [reference an ARG within a Dockerfile](https://docs.docker.com/engine/reference/builder/#using-arg-variables).  Before this change {} was the convention we used to search for strings to replace prior to building the Dockerfile.  Your concern on spaces in not applicable as the FROM image cannot contain spaces.

Reply:
Got it. Thanks.